docs/KEY_MANAGEMENT.md

```
@@ -0,0 +1,103 @@
+        let keys = Keys::from_mnemonic_advanced(
+            &self.mnemonic,
+            None,
+            Some(account),
+            Some(trade_index as u32),
+            Some(0),
+        )?;
+        Ok(keys)
+    }
+```
+
+## Database Persistence
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)